main.go

```diff
@@ -1,67 +1,159 @@
+// File service API
+//
+// Documentation for creating a file request and downloading files using the API
+//
+//   Schemes: http, https
+//   BasePath: /
+//   Version: 1.0.0
+//   securityDefinitions:
+//     Bearer:
+//       type: apiKey
+//       name: Authorization
+//       in: header
+//
+//   Consumes:
+// 	  - application/json
+//
+//   Produces:
+//    - application/json
+//
+// swagger:meta
 package main
 
 import (
-	"context"
-	"github.com/gorilla/mux"
-	"github.com/ministryofjustice/opg-go-healthcheck/healthcheck"
-	"log"
-	"net/http"
-	"opg-file-service/handlers"
-	"opg-file-service/middleware"
-	"os"
-	"os/signal"
-	"time"
+    "context"
+    "github.com/gorilla/mux"
+    "github.com/ministryofjustice/opg-go-healthcheck/healthcheck"
+    "log"
+    "net/http"
+    "opg-file-service/handlers"
+    "opg-file-service/middleware"
+    "os"
+    "os/signal"
+    "time"
```

Comment:
The official recommendation is to use tabs instead of spaces in the code itself (I know the swagger comments need two space characters, I guess that's fine as not really classed as code)

It's probably best to leave the GoLand setting to use tabs so we don't end up in a mess.

Applies to other places where tabs have been replaced with spaces

Reply:
I've switched back to tabs and double checked that the annotations still work. All good 👍